src/applications/hca/components/FormPages/FinancialConfirmation.jsx

```diff
@@ -6,12 +6,11 @@ const DisabilityConfirmation = props => {
   const { goBack, goForward } = props;
 
   return (
-    <va-alert
+    <va-featured-content
```

Comment:
@longmd - FYSA, there's a PR out to update this component from `va-featured-content` to `va-summary-box`. - https://github.com/department-of-veterans-affairs/vets-website/pull/27945

It hasn't been merged in just yet, but it's out there. Just putting it on your radar.

Reply:
I had seen that name change in Storybook yesterday. I'm curious if there is going to be a crossover where they'll both be active and we'll be told to switch over? I looked to see how many apps were using `va-featured-content` and I think it was like 6-8 at the moment.

Reply:
Nevermind... I see that they are being renamed in the PR. I will hold merging all three of my apps until that PR gets merged and to take advantage of the name change.